conans/test/functional/generators/visual_studio_test.py

```diff
@@ -72,6 +72,8 @@ def package_info(self):
             from conans import ConanFile, MSBuild
             import os
             class HelloConan(ConanFile):
+                name = "Hello Conan"
```

Comment:
Spaces are not allowed in a package name, I would say this should fail.

Reply:
That is fully consistent with me not running the tests! I apologize.

I'm on Windows. And running `tox` at the project root yields:
```bat
C:\projects\conan>python -mvenv venv
...
C:\projects\conan>.\venv\Scripts\install tox
...
C:\projects\conan>.\venv\Scripts\pip install -e .
...
C:\projects\conan>tox -e full
GLOB sdist-make: C:\projects\conan\setup.py
full inst-nodeps: C:\projects\conan\.tox\.tmp\package\1\conan-1.30.0.dev0.zip
full installed: beautifulsoup4==4.9.1,bottle==0.12.18,certifi==2020.6.20,chardet==3.0.4,colorama==0.4.3,conan @ file:///C:/projects/conan/.tox/.tmp/package/1/conan-1.30.0.dev0.zip,deprecation==2.0.7,distro==1.5.0,fasteners==0.15,future==0.18.2,idna==2.10,Jinja2==2.11.2,MarkupSafe==1.1.1,mock==1.3.0,monotonic==1.5,node-semver==0.6.1,nose==1.3.7,packaging==20.4,parameterized==0.7.4,patch-ng==1.17.4,pbr==5.5.0,pluginbase==1.0.0,Pygments==2.6.1,PyJWT==1.7.1,pyparsing==2.4.7,python-dateutil==2.8.1,PyYAML==5.3.1,requests==2.24.0,six==1.15.0,soupsieve==2.0.1,tqdm==4.48.2,urllib3==1.25.10,waitress==1.4.4,WebOb==1.8.6,WebTest==2.0.35
full run-test-pre: PYTHONHASHSEED='196'
full run-test: commands[0] | nosetests

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
_______________________________________________________ summary _______________________________________________________
  full: commands succeeded
  congratulations :)
```
Which isn't very helpful. I'm on Python 3.8.4 ... and `tox` runs for a coworker who is on 3.7.2 ... figures :)

Reply:
Having more luck w/ Python 3.7.9, it appears to be doing something at least. Unsure if it's a Python 3.8 ism or just a "I'm bad about using venvs and boogered up my Python 3.8 install".

Reply:
Note to future self, I couldn't get `tox` to find my compiler on Windows, so I had to follow these instructions:
https://github.com/conan-io/conan#without-tox

```sh
$ python -mvenv venv
$ ./venv/Scripts/pip install -e .
$ ./venv/python -m pip install -r conans/requirements.txt
$ ./venv/python -m pip install -r conans/requirements_server.txt
$ ./venv/python -m pip install -r conans/requirements_dev.txt
```

```sh
$ export CONAN_COMPILER="Visual Studio"
$ export CONAN_COMPILER_VERSION=16
```

And use `nose` instead.
```sh
$ ./venv/Scripts/python.exe -m nose -l DEBUG -v conans/test/functional/generators/visual_studio_test.py
```